### DISCUSSION TITLE
Downgrade version to latest which support PHP 8.0

### DISCUSSION THREAD

**Discussion by imakeitworknet** [statamic/cms#6039] [Troubleshooting] [answered]
Hi, made a silly mistake. Built a multisite (multilanguage) Statamic website for a client using latest Statamic version 3.3.11. When I needed to move site to customer's server realized that it only supports up to PHP 8.0 . Now composer is not happy, and site does not work.

Which latest Statamic version supports PHP 8.0? Any tips to downgrade site now without rebuilding it manually?

Thanks!

**Comment by duncanmcclean** [comment]
Statamic 3.3 does support PHP 8.0

**Reply by duncanmcclean** [reply]
What error are you getting?

**Reply by imakeitworknet** [reply]
Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.0.2". , But it shows that client's (shared) server runs 8.0.1..

I've built site on VPS with PHP 8.1 and it worked fine.

**Reply by edalzell** [reply]
they don't have a current version of php 8?

**Comment by jasonvarga** [comment, marked as answer]
Try adding `"php": "^8.0.1"` to your composer.json's `require` block and run `composer update`. I think that might lower the requirement down low enough so that it'll install on your server.

**Reply by imakeitworknet** [reply]
Just tried it but still the same error.

Also tried to cheat - disabled Composer dependency errors. Now the site runs - both client side, and admin.
But there is still an issue - eveywhere where I used glide for images I get

"The Process class relies on proc_open, which is not available on your PHP installation." so glide images won't work :/

**Reply by imakeitworknet** [reply]
Also spoke with server admins - they're going to update PHP version "in a few weeks". If there would be a way to temporary fix Glide I think it would be fine until server update.

**Reply by imakeitworknet** [reply]
Just figured out that it's not the Statamic issue - one of the addons which compresses images throws that error. Removed the addon, and everything works just fine in hared hosting and PHP 8.0.1. 

Thanks!
